test/components/elements/AutoCompleteMenuTest.js

```diff
@@ -0,0 +1,227 @@
+'use strict'
+import chai from 'chai'
+import spies from 'chai-spies'
+chai.use(spies)
+const expect = chai.expect
+
+import React from 'react'
+import AutoComplete from 'material-ui/AutoComplete'
+
+import createComponent from 'test/helpers/shallowRenderHelper'
+import AutoCompleteMenu, { onNewRequest } from 'components/elements/AutoCompleteMenu'
+import AddMockDispatch from 'test/helpers/mockDispatch'
+
+
+const mockProps = {
+    label: 'testLabel',
+    dataConfigText: 'textInput',
+    dataConfigValue: 'valueInput',
+    menuOptions: ['TEST1', 'TEST2'],
+    searchText: '',
+    onUpdateInput: () => null,
+    selectMethod: () => 42
+}
+
+const setup = (mockProps) => {
+
+    return createComponent(AutoCompleteMenu, mockProps)
+}
+
+describe('AutoCompleteMenu', function() {
+  describe('Component', function() {
+    describe('when it receives all required information', function () {
+      beforeEach(function () {
+          this.output = setup(mockProps)
+      })
+
+      it('should return a div', function () {
+          expect(this.output.type).to.equal('div')
+      })
+
+      it('the div should contain an AutoComplete', function () {
+          expect(this.output.props.children.type).to.equal(AutoComplete)
+      })
+
+      it('the Autocomplete should have the label we passed in', function () {
```

Comment:
maybe reword to `it('should.........?` and put the `'Autocomplete'` in a `describe()`